os2forms_forloeb_profile.info.yml

```diff
@@ -120,4 +120,3 @@ themes:
   - gin
   - claro
   - bootstrap
```

Comment:
Her kan vi fjerne bootstrap tematet også, da det er en dependency for custom_theme temaet.